backend/app/assets/javascripts/spree/backend/turbolinks-configuration.js.erb

```diff
@@ -0,0 +1,20 @@
+// comment line necessary for correct interpolation of use_turbolinks
+Turbolinks.supported = <%= Spree::Backend::Config.use_turbolinks %>;
+
+// override jQuery.ready to support Turbolinks
```

Comment:
This comment is misleading. You are doing this because we want to print out a deprecation warning and use `Spree.ready` even it it was not used explicitly. Turbolinks works with our this change.